lib/cmd/fh2/undeploy.js

```diff
@@ -1,8 +1,9 @@
+/* globals i18n */
 module.exports = undeploy;
 
-undeploy.desc = "Undeploy a cloud app";
-undeploy.usage = "fhc undeploy <app-id> [apptype] --env=<environment> (where 'apptype' is 'cloud' (default) or 'embed')" +
-  "\ne.g. fhc undeploy 12345 cloud --env=dev";
+undeploy.desc = i18n._("Undeploy a cloud app");
+undeploy.usage = i18n._("fhc undeploy <app-id> [apptype] --env=<environment> (where 'apptype' is 'cloud' (default) or 'embed')" +
+                       	"\ne.g. fhc undeploy 12345 cloud --env=dev");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace all tab characters in this file by sequences of white-spaces. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATabCharacter)